certora/harness/MarketplaceHarness.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.8;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
I'd prefer we use named imports, but to stay consistent with the rest of the codebase, i think we can leave as is also